api/envoy/extensions/filters/listener/http_inspector/v3/http_inspector.proto

```diff
@@ -18,4 +20,11 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message HttpInspector {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.listener.http_inspector.v2.HttpInspector";
+
+  // The number of bytes of the request the filter will initially buffer and
+  // inspect to determine the http version, the filter will iteratively double
+  // the buffer up to 64KiB. If not specified, the initial buffer is 8KiB.
+  google.protobuf.UInt32Value initial_read_buffer_size = 1
+      [(validate.rules).uint32 = {lt: 65537 gt: 16}];
+
```

Comment:
Why not just make this a field to set the maxReadBytes() that no larger than 64k? (with default value 8kb)